### PR TITLE
Bug 2024750: Fix OLM files in need of i18n

### PR DESCRIPTION
**Pull request by rebeccaalpert**
A few files needed i18n still.

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=2024750.

### CODE REVIEW

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by rebeccaalpert** [comment]
Putting this on hold until Dave's PR merges and fixes the e2e tests.

**Pull request merged**
